test/e2e/wasms/README.md

```diff
@@ -1,16 +1,13 @@
-Copy-pasted `.wasm` files (for now!)
-====================================
+Wasm files for testing
+======================
 
-These are built versions of contracts with source code that currently live in
-the `test-wasms` directory within soroban-tools. Even there, they are mostly
-copy-pasted contracts from `soroban-examples`.
+These are copies of Wasm files from testnet or other live networks, used for testing
 
-It would probably be best to include `soroban-examples` as a git submodule,
-then directly rely on the contracts we want to test against. No more
-copy-pasting! But until we can all agree on such an approach, maybe this is
-good enough?
+The two files here were created with:
 
-Soon the CLI will have a `soroban init` command that allows setting up a new
-Cargo workspace and may allow specifying which example contracts to put in it.
-If the git submodule is controversial, then we could use that new `init`
-command instead.
+```
+stellar contract fetch --id CA4SAAVG73IW6JQK4F45NAR7IAT7DI5MQPRSYSQWEWX4RKUNLL77MNOF --network testnet --out-file test/e2e/wasms/stellar_asset_wrapper.wasm
```

Comment:
BTW, this contracts are not available on the testnet anymore, so I don't think mentioning them here worth it